Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -13,22 +13,17 @@ extension SPNoteListViewController {
         registerForPreviewing(with: self, sourceView: tableView)
     }
 
-    /// Refreshes the SearchBar Style.
+    /// Refreshes the ListViewController's Title
     ///
     @objc
-    func styleSearchBar() {
-        let searchIconColor = UIColor.color(name: .simplenoteSlateGrey)
-        let searchIconImage = UIImage.image(name: .searchIconImage)?.withOverlayColor(searchIconColor)
+    func refreshTitle() {
+        let selectedTag = SPAppDelegate.shared().selectedTag ?? NSLocalizedString("All Notes", comment: "Title: No filters applied")
 
-        searchBar.setImage(searchIconImage, for: .search, state: .normal)
-        searchBar.setSearchFieldBackgroundImage(.searchBarBackgroundImage, for: .normal)
-        searchBarContainer.backgroundColor = .clear
-
-        // Apply font to search field by traversing subviews
-        for textField in searchBar.subviewsOfType(UITextField.self) {
-            textField.font = .preferredFont(forTextStyle: .body)
-            textField.textColor = .color(name: .textColor)
-            textField.keyboardAppearance = SPUserInterface.isDark ? .dark : .default
+        switch selectedTag {
+        case kSimplenoteTagTrashKey:
+            title = NSLocalizedString("Trash-noun", comment: "Title: Trash Tag is selected")
```

Comment:
would I assume correctly if I were to say that this is `Trash-noun` but above we have `All Notes` because the translations have already been built based on those things, and that's the reason to leave them instead of update `All Notes-noun`? Or is it that `All notes` seems less ambiguous than `Trash` and that's why they are different?

Reply:
I've kept the string this way, because the translations are already in place. You would never see `Trash-noun` in the actual UI (as long as there's an entry [here](https://github.com/Automattic/simplenote-ios/blob/develop/Simplenote/en.lproj/Localizable.strings)... or in the appropriate language).

**NSLocalizedString** takes the first parameter, and picks up the appropriate `Localizable.strings` file, based on the system language. Then, the target string is picked up.

In this case, `Trash-noun` [maps here](https://github.com/Automattic/simplenote-ios/blob/develop/Simplenote/en.lproj/Localizable.strings#L218), and you get `Trash` onscreen.

Reply:
thanks. I had assumed it was because they already existed.